src/realm.h

```diff
@@ -1942,7 +1942,8 @@ RLM_API size_t realm_object_changes_get_modified_properties(const realm_object_c
  */
 RLM_API void realm_collection_changes_get_num_changes(const realm_collection_changes_t*, size_t* out_num_deletions,
```

Comment:
We need to update the documentation as well. Can you add the argument above in the comment pls?